modules/configuring-firewall.adoc

```diff
@@ -101,6 +101,10 @@ You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn
 |Required to access Alibaba Cloud services and resources. Review the link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/sdk/endpoints/endpoints_config.go?spm=a2c4g.11186623.0.0.47875873ciGnC8&file=endpoints_config.go[Alibaba endpoints_config.go file] to determine the exact endpoints to allow for the regions that you use.
 
 .15+|AWS
+|`aws.amazon.com`
+|443,80
```

Comment:
Looks like other spots in this table are putting a space in there:
```suggestion
|443, 80
```

Reply:
Good catch, thanks